packages/stan/README.md

```diff
@@ -0,0 +1,203 @@
+-   Supports async and sync state.
+
+Unlike these frameworks, it has the following goals too: (which justified the creation of a separate library)
+
+-   It is React independent. You can create binding for any of your desired framework.
+-   It needs to be flexible enough to offer bindings for `@wordpress/data` consumer API. (useSelect and useDispatch).
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/stan --save
+```
+
+_This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
+
+## Getting started
+
+Stan is based on the concept of "atoms". Atoms are discrete state units and your application state is a tree of atoms that depend on each other.
+
+### Creating basic atoms
+
+Let's create some basic atoms:
+
+```js
+import { createAtom } from '@wordpress/stan';
+
+// Creates an atom with an initial value of 1.
+// The value can be of any type.
+const counter = createAtom( 1 );
+```
+
+### Manipulating atoms
+
+In this example we created an atom that can hold a counter that starts with `1`. 
+To manipulate we need a registry. The registry is the container of all atom states.
+
+```js
+import { createAtomRegistry } from '@wordpress/stan';
+
+const registry = createAtomRegistry();
+
+// Read the counter.
+console.log( registry.get( counter ) ); //  prints 1.
+
+// Modify the value of the counter
+registry.set( counter, 10 );
+
+console.log( registry.get( counter ) ); //  prints 10.
+```
+
+### Subscribing to changes
+
+Each atom is an observable to which we can subscribe:
+
+```js
+registry.subscribe( counter, () => {
+    console.log( registry.get( counter ) );
+} );
+
+registry.set( counter, 2 ); // prints 2.
+registry.set( counter, 4 ); // prints 4.
+```
+
+### Derived atoms
+
+Atoms can also derive their value based on other atoms. We call these "derived atoms".
+
+```js
+import { createAtom, createDerivedAtom } from '@wordpress/stan';
+
+const counter1 = createAtom( 1 );
+const counter2 = createAtom( 2 );
+const sum = createDerivedAtom(
+    ( { get } ) => get( counter1 ) + get( counter2 )
+);
+```
+
+In the example above, we create two simple counter atoms and third derived "sum" atom which value is the sum of both counters.
+
+```js
+console.log( registry.get( sum ) ); // prints 3.
+
+// Adding a listener automatically triggers the refreshing of the value.
+// If the atom has no subscriber, it will only attempt a resolution when initially read. 
+// But it won't bother refreshing its value, if any of its dependencies change.
+// This property (laziness) is important for performance reasons.
+sumInstance.subscribe( () => {
```

Comment:
What is `sumInstance` here?

Reply:
good catch, it's something I decided to hide inside the registry (atom state objects)